commit 42e9986d0d46b798aaa9113b0cd8b009e5a62598
Author: freedesktop_sdk_updater <freedesktop_sdk_updater@libreml.com>
Date:   Fri Jan 24 14:34:29 2025 +0000

    Update elements/include/linux.yml to v6.13-0
    
    Updates elements/include/linux.yml to v6.13-0 (ffd294d)
    Previous version was v6.12.9-0 (5996393)

diff --git b/elements/include/linux.yml a/elements/include/linux.yml
index 05dff1f9f..249b0059a 100644
--- b/elements/include/linux.yml
+++ a/elements/include/linux.yml
@@ -4,7 +4,7 @@ sources:
   track: v*
   exclude:
   - '*-rc*'
-  ref: v6.13-0-gffd294d346d185b70e28b1a28abe367bbfe53c04
+  ref: v6.12.9-0-g5996393469d99560b7845d22c9eff00661de0724
 - kind: local
   path: files/linux/config-utils.sh
 - kind: local
